packages/connections/src/components/connection-list/connection.tsx

```diff
@@ -181,6 +182,9 @@ function Connection({
           </Description>
         )}
       </button>
+      <ConnectionMenu
+        connectionString={connectionInfo.connectionOptions.connectionString}
+      />
```

Comment:
Would it make sense to only show it on focus/hover so the element is not repeated too much in the list? It feel like the current way of showing it creates a lot of visual noise in the list that we probably can avoid here